user/hashcat/template.py

```diff
@@ -0,0 +1,34 @@
+pkgname = "hashcat"
+pkgver = "6.2.6"
+pkgrel = 0
+build_style = "makefile"
+make_env = {
+    "PREFIX": "/usr",
+    # needs LzmaDec.h
+    # "USE_SYSTEM_LZMA": "1",
```

Comment:
this is the 7zip-sdk, not related to xz-devel, so this would never be system, just remove the comment